Cakes-Co/css/style.css

```diff
@@ -0,0 +1,312 @@
+
+  --default-font-family: "Nunito Sans", sans-serif;
+  --heading-font-family: "Playfair Display", serif;
+
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--darker-pink);
+  font-family: var(--default-font-family);
+}
+
+.container {
+  margin: 0 auto;
+  position: relative;
+}
+
+a {
+  text-decoration: none;
+  color: var(--gold);
+  transition: all 0.2s ease-in-out;
+}
+
+li {
+  list-style: none;
+}
+
+h1 {
+  font-family: var(--heading-font-family);
+  margin: 0 0 0.5rem 0;
+}
+
+h3 {
+  font-weight: 100;
+  font-style: italic;
+  align-self: flex-end;
+}
+
+/* Header */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  gap: 1rem;
+  align-items: center;
+  background-color: var(--light-pink);
+  padding: 1rem;
+}
+
+.logo {
+  max-width: 100px;
+}
+
+.branding {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.branding p {
+  font-family: var(--heading-font-family);
+  font-size: 1.4rem;
+  /* margin-top: -20px; */
+  /* text-shadow: 0 -1px var(--gold); */
+  color: var(--gold);
+}
+
+#menu {
+  display: none;
+  text-align: center;
+}
+
+.menu_item {
+  display: block;
+  background-color: var(--light-pink);
+  padding: 1rem 2rem;
+}
+.menu_item:hover {
+  background-color: var(--darker-pink);
+  color: var(--white);
+}
+
+.icon {
+  display: block;
+  position: absolute;
+  right: 10px;
+  top: 10px;
+  font-size: 32px;
+}
+
+/* medium */
+@media (min-width: 768px) {
+  #menu {
+    display: flex!important;
+    justify-content: center;
+    gap: 1rem;
+    margin-top: 0.2rem;
+    background-color: var(--light-pink);
+  }
+  .icon {
+    display: none;
+  }
+}
+
+/* large */
+@media (min-width: 1024px) {
+  .header {
+    height: 150px;
+    padding: 2rem 3rem;
+  }
+
+  .logo {
+    max-width: 170px;
+  }
+  nav {
+    display: flex;
+    justify-content: center;
+  }
+
+  #menu {
+    width: 90%;
+    justify-content: right;
+    padding: 1rem;
+    gap: 0.5rem;
+    margin: 1.5rem 4rem;
+    flex-grow: 1;
+  }
+  .menu_item {
+    color: var(--white);
+    background-color: var(--darker-pink);
+  }
+  .menu_item:hover {
+    background-color: var(--light-pink);
+    color: var(--gold);
+  }
+}
+
+/* Main Styles */
+
+main {
+  padding: 1.25rem;
+}
+
+.welcome {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+
+.cupcake-group {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  gap: 1rem;
+  margin-bottom: 2rem;
+}
+
+.cupcake-container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.middle-btn {
+  transition: 0.5s ease;
+  opacity: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+}
+
+.cupcake_item-img {
+  transition: 0.5s ease;
+  opacity: 1;
+  width: 100%;
+}
+
+.cupcake-container:hover .cupcake_item-img {
```

Comment:
I love the hover styles with the buttons, I think they're a really nice touch, especially how seemless they feel